.github/workflows/autoremove_trivyignore.yaml

```diff
@@ -0,0 +1,57 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+name: Auto-remove ignored CVEs
+on:
+  workflow_call:
+
+jobs:
+  update-lib:
```

Comment:
Can you rename this job?

Reply:
I will remove this job, we will be annotating the trivyignore based on CVEs found in the rock and that job is no longer needed, thanks for spotting anyway!